app/views/media_objects/_item_view.html.erb

```diff
@@ -115,6 +117,63 @@ Unless required by applicable law or agreed to in writing, software distributed
       }
     });
     currentPlayer = avalonPlayer.player;
+
+    var $video = currentPlayer.container;
+    var $controls = $video.find('.mejs-controls');
+    var controls_hidden = false;
+    setup_controls_autohide()
+
+    function setup_controls_autohide() {
+      setup_controls_autohide_by_mouse_position();
+      setup_controls_autohide_by_mouse_movement();
+    }
+
+    function setup_controls_autohide_by_mouse_position() {
+      // Mouse enters video region: controls appear
+      // Mouse leaves video region: controls hide
+      $video.mouseleave(function() {
```

Comment:
Just testing with the Chrome mobile emulation, looks to work okay on mobile.
The mouse move stuff doesn't work, but a decent autohide behaviour is still captured by the timer (controls disappear two seconds after the last touch).
Would love to test on a real mobile device, but the UNIX team have blocked the test machine from UWS traffic ...